cloud-api-reference/openapi.yml

```diff
@@ -53,92 +53,33 @@ paths:
           description: Internal server error
       security:
       - bearerAuth: []
-      x-codeSamples:
```

Comment:
Why have the `x-codeSamples` been removed across this file?

Reply:
Great observation, added it back